src/devices/scmplus.c

```diff
@@ -0,0 +1,198 @@
+/** @file
+    ERT SCM+ sensors.
+
+    Copyright (C) 2020 Peter Shipley <peter.shipley@gmail.com>
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+*/
+
+#include <arpa/inet.h>
+#include "decoder.h"
+
+/**
+
+Freq 912600155
+
+Random information:
+
+https://github.com/bemasher/rtlamr/wiki/Protocol
+http://www.gridinsight.com/community/documentation/itron-ert-technology/
+
+ Units: Some meter types transmit consumption in 1 kWh units, while others use more granular 10 Wh units
```

Comment:
Is there no way to know this ?

Reply:
not that i can find.  rtlamr has the same comment